internal/cmdutil/cmdutil.go

```diff
@@ -170,3 +172,49 @@ func MustGetProof(path string) delegation.Delegation {
 	}
 	return proof
 }
+
+// ParseSize parses a data size string with optional suffix (B, K, M, G).
+// Accepts formats like: "1024", "512B", "100K", "50M", "2G". Digits with no
+// suffix are interpreted as bytes. Returns the size in bytes.
+func ParseSize(s string) (uint64, error) {
```

Comment:
https://github.com/dustin/go-humanize us nice for this.

Reply:
I'm using that in the UI, but it doesn't do *parsing*, does it?